src/injected/content/inject.js

```diff
@@ -8,14 +8,16 @@ import {
 import { sendCmd } from '#/common';
 
 import {
-  forEach, join, append, createElementNS, NS_HTML,
+  forEach, join, append, createElementNS, defineProperty, NS_HTML,
   charCodeAt, fromCharCode,
 } from '../utils/helpers';
 import bridge from './bridge';
 
 // Firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1408996
-const VMInitInjection = window[process.env.INIT_FUNC_NAME];
-delete window[process.env.INIT_FUNC_NAME];
+const VMInitInjection = window[Symbol.for(process.env.INIT_FUNC_NAME)];
+// To avoid running repeatedly due to new `document.documentElement`
+// (the symbol is undeletable so a userscript can't fool us on reinjection)
+defineProperty(window, Symbol.for(process.env.INIT_FUNC_NAME), { value: 1 });
```

Comment:
What's the benefit of using a symbol instead of the string, especially when the symbol is created in a global list, i.e. userscripts can also get it by `Symbol.for`?

Reply:
The benefit is that it's non-enumerable by default so I don't have to use Object.defineProperty which may be hacked by a content userscript on reinjection of documentElement.

Reply:
To clarify: I would have to use a compromised Object.defineProperty in webpack.conf:

```js
      new WrapperWebpackPlugin({
        header: `if (window[Symbol.for('${INIT_FUNC_NAME}')]!==1)
          window[Symbol.for('${INIT_FUNC_NAME}')] = function () {
            var module = { exports: {} };
          `,
```

Reply:
Another benefit is that it won't clash with user keys or symbols when those are used in the intended, standard fashion (as for the ability of userscripts to access it using Symbol.for, this won't help them because the symbol will non-configurable after first init).

Reply:
I am convinced.

Reply:
Actually, the **only** benefit is that it won't clash with user keys/symbols. This is an important benefit though.